ustriage/ustriage.py

```diff
@@ -806,7 +841,14 @@ def launch():
                         type=int,
                         dest='age',
                         help='Mark bugs touched more recently than this many'
-                             ' days (default disabled in triage, 7 days in '
+                             ' days (default disabled in triage, 6 days in '
+                             ' tag/subscription search)')
+    parser.add_argument('--flag-old',
+                        default=False,
+                        type=int,
+                        dest='old',
+                        help='Mark bugs not touched for this many days'
+                             ' (default disabled in triage, 90 days in '
```

Comment:
In this chunk two hardcoded defaults are mentioned in the help strings (`6 days` and `90 days`). I fear the help screen will go out of sync with the actual value at some point. Can we add some defaults (e.g. `FLAG_RECENT_AGE` and `FLAG_OLD_AGE`) to be referenced here and in code?

Reply:
I totally agree ... added